include/libkdumpfile/addrxlat.h.in

```diff
@@ -829,6 +829,11 @@ typedef enum _addrxlat_optidx {
 	 */
 	ADDRXLAT_OPT_rootpgt,
 
+	/** User land root page table address. Used *only* in aarch64 case.
+	 * Type: full address
```

Comment:
The comment is confusing. I don't see anything architecture-specific here, and in fact the option should also be very useful on IBM POWER.